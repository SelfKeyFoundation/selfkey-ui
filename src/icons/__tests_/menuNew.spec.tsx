import * as React from 'react';
import { create } from 'react-test-renderer';
import MenuNewIcon from '../menu-new';

describe('Menu New Icon', () => {
  it('should show Menu New Icon', () => {
    const component = create(
      <MenuNewIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});