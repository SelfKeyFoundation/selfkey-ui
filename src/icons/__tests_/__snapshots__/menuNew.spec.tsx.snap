// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu New Icon should show Menu New Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1"
  focusable="false"
  style={
    Object {
      "height": "21px",
      "width": "30px",
    }
  }
  viewBox="0 0 30 21"
>
  <g
    id="MD-mn"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="#00C0D9"
      fillRule="nonzero"
      id="300.4-Nc-mn"
      transform="translate(-76.000000, -27.000000)"
    >
      <path
        d="M76,27 L76,28.96 L101,28.96 L101,27 L76,27 Z M76,36.52 L76,38.48 L106,38.48 L106,36.52 L76,36.52 Z M76,46.04 L76,48 L96,48 L96,46.04 L76,46.04 Z"
        id="icon-menu-new-mn"
        stroke="transparent"
        strokeWidth="10"
      />
      <polygon
        id="Path"
        points="220 36.52 220 38.48 250 38.48 250 36.52"
      />
    </g>
  </g>
</svg>
`;
